doi-requests-commons/src/test/java/no/unit/nva/doi/requests/userdetails/UserDetailsTest.java

```diff
@@ -0,0 +1,45 @@
+package no.unit.nva.doi.requests.userdetails;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.nio.file.Path;
+import nva.commons.handlers.RequestInfo;
+import nva.commons.utils.IoUtils;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class UserDetailsTest {
+
+    private static final String FEIDE_ID_IN_FILE = "feide@id";
+    private static final String CUSTOMER_ID_IN_FILE = "123456";
+    private static final String ROLES_IN_FILE = "Role1,Role2,Role3";
+    private RequestInfo requestInfo;
+
+    @BeforeEach
+    public void init() throws JsonProcessingException {
```

Comment:
Tests for code coverage. Bad case scenarios were not implemented as the  methods perform only a library call.